### ISSUE TITLE
What is the use case for this image?

### ISSUE THREAD

**Issue by rastasheep** [rastasheep/ubuntu-sshd#6]
Hi all!

Thanks for using my image.
This is an old (forgotten) side project which wasnt updated in awhile, but with help of automatic builds on docker hub, it's still up to date and it seems that it is used widely (currently it was downloaded more than 100k times).

Can you share your use cases for this kind of image? I would like to invest some time in improvement and those informations would be more than useful to me, tnx.

**Comment by ncs-alane** [comment]
It's useful for integration testing of configuration management logic!

**Comment by rastasheep** [comment]
@ncs-alane tnx, interesting i've always used vagrant for testhing CM stuff :)

**Comment by KhumpuW** [comment]
I use it for learning Docker.

**Comment by snikobonyadrad** [comment]
Really thanks for this. I was looking for a quick way to test ansible deployment and this was fantastic.

**Comment by Foorack** [comment]
@rastasheep We are currently using it as a connector for ssh tunnels. We are currently using a modified version, it would be very helpful if GatewayPorts was enabled, or if there at least were a way to toggle this. :+1:

**Comment by rastasheep** [comment]
@Foorack tnx :) It sounds like modifying configuration in runtime with env variables would solve your use-case

**Comment by Foorack** [comment]
@rastasheep Yes, it would. Currently, we are having to build our own version with an extra line in the Dockerfile enabling GatewayPorts. That would be a great improvement!

**Comment by rastasheep** [comment]
@Foorack awesome, i've created an issue so if anyone wants to help it's good to go .. tnx again for the feedback!

**Comment by joncamfield** [comment]
I'm interested in having a dockerized sshd on my synology, ideally one where I can:

* customize ports (optional, can do via port fowarding)
* create and restrict ssh users (create sudoer user, disable remote root login) 
* require ssh key login
* have a slightly more configurable "landing" shell (for ssh tunneling, simple bash / network testing tools, quick-and-dirty SCP'ing of files)

**Comment by RamonSmit** [comment]
I'm using this in our Kubernetes cluster, just for a quick and dirty easy access to persistent volumes via sftp. Works really nice for the 5 min jobs 👍

**Comment by dagguh** [comment]
I'm using it for testing SSH scripts targeting Ubuntu.
